src/ol/renderer/webgl/worldUtil.js

```diff
@@ -0,0 +1,29 @@
+import {getWidth} from '../../extent.js';
+
+/**
+ * Compute world params
+ * @private
```

Comment:
I'm unclear on what the impact of a `@private` annotation on an exported function is.  Does this mean other modules cannot or should not import?